BUILD_MESON

```diff
@@ -209,7 +208,7 @@ Why we use another option for sbindir?
 
 For some reasons, Old Mesons force sbindir will prefixed by Meson's prefix
 (eg /usr + sbin) but Dinit project use /sbin by default as sbindir.
-in Newer Mesons that fixes this thing but we target Mesons's 0.47.0 as
+in Newer Mesons that fixes this thing but we target Mesons's 0.56.0 as
```

Comment:
What version of Meson is this actually fixed in?

Reply:
https://mesonbuild.com/Release-notes-for-0-62-0.html#all-directory-options-now-support-paths-outside-of-prefix

Reply:
Hmm, it's a bit werid but no one talks about that issue and its fix in release notes (or just I didn't see something related to that)! related fix merged in 9 Feb 2022 so it's fixed somewhere at release 0.6x.x.

Reply:
Can we change the note here to explicitly mention the version (0.62.0 according to Eli's post)?

Reply:
Looks ok, I will do that.